scripts/changelog.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+
```

Comment:
Could we include a short helper to outline the expected parameters here?
Something like
`changelog.py <old-release-tag> <new-release-tag>`

Reply:
There is help command but I added a comment with example